test/powershell/Language/Scripting/NativeExecution/NativeCommandProcessor.Tests.ps1

```diff
@@ -140,6 +140,10 @@ Describe "Native Command Processor" -tags "Feature" {
             [Console]::OutputEncoding = $originalOutputEncoding
         }
     }
+
+    It '$ErrorActionPreference does not apply to redirected stderr output' {
+        pwsh -noprofile -command "`$ErrorActionPreference = 'Stop'; testexe -stderr stop 2>`$null; 'hello'" | Should -BeExactly 'hello'
```

Comment:
```suggestion
        pwsh -noprofile -command '$ErrorActionPreference = "Stop"; testexe -stderr stop 2>$null; "hello"' | Should -BeExactly 'hello'
```

Swapping the quotation marks should be safe since PowerShell is the active shell here, and it avoids needing escape characters.

Reply:
Will change

Reply:
It's odd, with the double quotes inside, the exe gets args where the double quotes get removed.  I validated this with testexe and also without my changes.  Not sure what's going on there, but it's not related to the changes in this PR.